src/javascript/ContentManager/actions/zipUnzip/unzipAction.jsx

```diff
@@ -0,0 +1,23 @@
+import {composeActions} from '@jahia/react-material';
+import requirementsAction from '../requirementsAction';
+import {withNotificationContextAction} from '../withNotificationContextAction';
+import * as _ from 'lodash';
+import {map} from 'rxjs/operators';
+import zipUnzipMutations from './zipUnzip.gql-mutations';
+import {refetchContentTreeAndListData} from '../../ContentManager.refetches';
+
+export default composeActions(requirementsAction, withNotificationContextAction, {
+    init: context => {
+        context.initRequirements({
+            retrieveMimeType: true,
+            enabled: context => context.node.pipe(map(node => !_.isEmpty(node.children.nodes) && node.children.nodes[0].mimeType.value === 'application/zip'))
```

Comment:
Could we get rid of lodash here? Seems to be the only reason for importing it in the first place.